bin/aomp_common_vars

```diff
@@ -245,6 +237,24 @@ if [ "$AOMP_APPLY_ROCM_PATCHES" == 1 ] ; then
 fi
 }
 
+function removepatch(){
+if [ "$AOMP_APPLY_ROCM_PATCHES" == 1 ] && [ "AOMP_STANDALONE_BUILD != 1" ] ; then
```

Comment:
[ "AOMP_STANDALONE_BUILD != 1" ] needs to be changed to [ "$AOMP_STANDALONE_BUILD" != 1]
While this did not affect the ROCM build it will still execute the remove patch on the stand alone, which we do not want.